src/inc/quic_datapath.h

```diff
@@ -205,7 +205,7 @@ CxPlatDataPathRecvPacketToRecvData(
 //
 CXPLAT_RECV_PACKET*
 CxPlatDataPathRecvDataToRecvPacket(
-    _In_ const CXPLAT_RECV_DATA* const RecvData
```

Comment:
For this file, the names are all correct. If the Linux datapath file is out of sync (and this clang is complaining) then that is where the fixes should be.

Reply:
We recently did a lot of refactoring and I only ensured the header was correct and didn't have a chance to go fix the implementations yet. Feel free to just fix the absolute minimum to make this happy.